Tests/StaticResources/Dialect3/Valid/1_box.uif

```diff
@@ -0,0 +1,112 @@
+// A representation of a 1-box POETS Engine in Dialect 3. Costs are unknown.
```

Comment:
do i recall that you were going to add my MPI backend uif to the tests for this as well?

Reply:
Yes - I said I'd do that. I have an older version on Slack, but could you punt me your current one so I can track it?

Reply:
It seems I did do this after all - it's in `valid_dialect_3_one_thread_per_core.uif`.